tasmota/xsns_69_opentherm_protocol.ino

```diff
@@ -431,6 +441,32 @@ void sns_opentherm_tele_oem_diag(struct OpenThermCommandT *self)
     ResponseAppend_P(PSTR("%d"), (int)self->m_results[0].m_u16);
 }
 
+/////////////////////////////////// Boiler Boiler Lock-out Reset  //////////////////////////////////////////////////
+unsigned long sns_opentherm_send_blor(struct OpenThermCommandT *self, struct OT_BOILER_STATUS_T *status)
+{
+    AddLog(LOG_LEVEL_ERROR, PSTR("[OTH]: Call Boiler Lock-out Reset"));
+    
+    self->m_flags.skip = true; // Disable future calls of this command
```

Comment:
As I mentioned earlier, in case of OT comm failure your BLOR command will not be delivered to the boiler. If you want to keep it as-is, it's okay. Just make sure your logic will periodically make attempts to make an `ot_blor` calls.

Reply:
I think that only manual controlled call of BLUR is valid. You should never use this command in automation scripts. I don't know any commercial OT adapters that support BLOR. Support team usual answer that you should't remotely reset boiler....